### PR TITLE
SID Offshore @ KSFO and Spawn Strips on Top

### PR DESCRIPTION
**Pull request by sixeyeco**
ksfo.json addresses cfit described in https://github.com/zlsa/atc/issues/324.
aircraft.js flips the strip order as described in https://github.com/zlsa/atc/issues/340

(i hope you're able to ignore the "delete-this" commits)

### CODE REVIEW

**Comment by erikquinn** [comment]
I like the top-down order! :smile: Is it possible to get the stack of strips to stay at the bottom of the screen instead of the top? If not, no biggie.

Also, maybe it would be nice to have a barely visible line enclosing the strip so it has the look and feel of a physical object? I'm not sure if it looks good though, it may be cleaner to not do what I'm saying-- just thinking aloud here is all.

Trying it myself, I added the following 6 lines to `strip.css`:

```
.strip {
  background-color: rgba(255, 255, 255, 0.0);
  border-width: 0px 4px 0px 4px;

  border-left: 4px solid transparent; 
  border-right: 4px solid transparent; 
+  border-top: 1px solid transparent; 
+  border-bottom: 1px solid transparent; 

  list-style-type: none;
  position: relative;
  padding: 3px 10px;
  margin-bottom: 5px;
  margin-left: 0;
  margin-right: -3px;

  transition: margin-left 0.1s ease, margin-right 0.1s ease, background-color 0.05s ease, box-shadow 0.05s ease;
}

.strip.arrival {
  border-color: rgba(224, 128, 128, 0.8);
+  border-top-color: rgba(224, 128, 128, 0.1);
+  border-bottom-color: rgba(224, 128, 128, 0.1);
}

.strip.departure {
  border-color: rgba(128, 255, 255, 0.8);
+  border-top-color: rgba(128, 255, 255, 0.05);
+  border-bottom-color: rgba(128, 255, 255, 0.05);
}
```

image

**Comment by sixeyeco** [comment]
<b>@everyone!</b> Sorry for all those css edits. I have no idea what I'm doing. I created a branch and I mistakenly made edits on the main fork. All apologies.

@erikquinn I really like what you've done. Perhaps I can close this one and you can pr yours?

**Comment by erikquinn** [comment]
@sixeyeco nah, its your work, you can get the credit. Feel free to merge mine into this PR you already have if you like :smile:

File: assets/airports/ksfo.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
       ["ups", 2]
     ],
     "sids": {
-      "OFFSHORE": ["SEPDY", "SENZY", "WAMMY", "SEGUL"],
+      "OFFSHORE": ["SENZY", "WAMMY", "SEGUL"],
```
Comment (erikquinn) [inline assets/airports/ksfo.json:117]: This is a good solution for now while work continues on #324. :+1:

**Comment by erikquinn** [comment]
The intent of this pull request has been carried out in #458. Since that has now been merged, this proposed change has been effected, and the request may be closed.

**Pull request closed without merging**